src/FSharp.Markdown/MarkdownParser.fs

```diff
@@ -449,6 +449,35 @@ let rec (|ListItems|_|) prevSimple = function
 
 // Code for parsing pipe tables
 
+// Splits table row into deliminated parts escaping inline code and latex
+let rec PipeTableFindSplits (delim : char array) (line : char list) = 
```

Comment:
You can just commit another commit and push it to the same branch you started the pull request with.